util/file/fileutil.go

```diff
@@ -63,23 +73,52 @@ func DecodeDecompressString(content string) (string, error) {
 	return string(dBuf), nil
 }
 
+type GZipWriter interface {
```

Comment:
isn't this the same interface as `io.Writer`?

Reply:
It's not since `io.Writer` does not implement `Close()`. However I just found `io.WriteCloser` which is exactly what I needed. Just switched.